projects/packages/videopress/src/client/admin/components/video-row/index.tsx

```diff
@@ -184,6 +186,12 @@ const VideoRow = ( {
 	const showTitleLabel = ! isSmall && isEllipsisActive;
 	const showStats = ( ! showActions && ! isSmall ) || ( isSmall && expanded );
 	const showBottom = ! isSmall || ( isSmall && expanded );
+	const canExpand =
+		isSmall &&
+		( ! hideEditButton ||
+			Boolean( duration ) ||
+			Boolean( plays ) ||
```

Comment:
`plays` can be 0, right? In this case, `Boolean( plays )` returns `false`. If you are only checking that there is a value and 0 is a valid value (which I'm not sure here), `plays != null` would return `true` for `0`.

Reply:
Done: https://github.com/Automattic/jetpack/pull/25968/commits/3a8f2a3df8692cff5fccb67ca3954150c6b773b6